src/dbnode/encoding/arrow/tile/recorder.go

```diff
@@ -63,6 +64,9 @@ func newDatapointRecorder(pool memory.Allocator) *datapointRecorder {
 		nil,
 	)
 
+	//float64s := array.NewFloat64Data(nil)
+	//float64s.Reset(data)
```

Comment:
@arnikola I've commented out this recent addition as it did not compile (and looks like it was a noop).
Might be some unfinished improvement - please have a look.